packages/devtools/lib/src/framework/framework.dart

```diff
@@ -19,6 +20,10 @@ import '../ui/ui_utils.dart';
 import '../utils.dart';
 import 'framework_core.dart';
 
+bool get importMode => _importMode;
```

Comment:
this should probably be in globals not framework. Also add a comment explaining that 
importMode is an offline mode operating on a imported data file.

Reply:
done.